website/content/docs/internals/telemetry/metrics/raft.mdx

```diff
@@ -166,4 +166,28 @@ Vault [integrated storage](/vault/docs/configuration/storage/raft).
 
 @include 'telemetry-metrics/vault/raft-storage/put.mdx'
 
-@include 'telemetry-metrics/vault/raft-storage/transaction.mdx'
\ No newline at end of file
+@include 'telemetry-metrics/vault/raft-storage/transaction.mdx'
+
+## raft-wal metrics
+
+@include 'telemetry-metrics/vault/raft-wal/head-truncations.mdx'
```

Comment:
It's interesting to me that we link these metrics in two places. I noticed we also do this for raft-storage metrics, so probably fine, but still interesting...